lib/cache/on-disk.ts

```diff
@@ -80,9 +82,14 @@ export class OnDiskCache extends BaseCache {
                 };
             })
             .filter(Boolean);
+
         // Sort oldest first
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore filter(Boolean) should have sufficed but doesn't
```

Comment:
wouldn't casting/setting the type of info to CacheEntry[] fix this?

Reply:
Afraid not, `sort` isn't a field of `CacheEntry`

Reply:
it's part of arrays (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)